services/spark-operator/1.2.3/spark-operator.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: spark-operator
+  namespace: ${releaseNamespace}
+spec:
+  chart:
+    spec:
+      chart: spark-operator
+      sourceRef:
+        kind: HelmRepository
+        name: mesosphere.github.io-spark-charts
+      version: 1.2.3
```

Comment:
for my reference, Is this the chart version? 

I only see `1.1.6` when looking at the chart repo:
```
NAME                	CHART VERSION	APP VERSION        	DESCRIPTION                                  
spark/spark-operator	1.1.6        	v1beta2-1.2.3-3.1.1	A Helm chart for Spark on Kubernetes operator
```

Reply:
Good catch! Fixed